.circleci/config.yml

```diff
@@ -55,6 +65,48 @@ jobs:
           paths:
             - project
 
+  #### cypress e2e tests
```

Comment:
@tomklapka Shouldn't we remove this, because we rely upon github workflows to execute e2e tests?

Reply:
I think that you can make it simple and completely remove the v2 stuff, v2 tests can remain in the original repository so that they could be eventually when required.

Reply:
ok